src/librbd/cache/pwl/AbstractWriteLog.h

```diff
@@ -360,7 +360,7 @@ class AbstractWriteLog {
 
   virtual void process_work() = 0;
   virtual void append_scheduled_ops(void) = 0;
-  virtual void schedule_append_ops(pwl::GenericLogOperations &ops) = 0;
+  virtual void schedule_append_ops(pwl::GenericLogOperations &ops, C_BlockIORequestT *req = nullptr) = 0;
```

Comment:
I don't think `req` needs to be defaulted here since both callers (`schedule_append()` overloads) pass it.